relay-event-normalization/src/event.rs

```diff
@@ -229,7 +229,7 @@ fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) {
     // Some contexts need to be normalized before metrics extraction takes place.
     normalize_contexts(&mut event.contexts);
 
-    if event.ty.value() == Some(&EventType::Transaction) && !config.is_renormalize {
```

Comment:
`normalize` is not called when not renormalization, so this check is redundant (see line 141). I didn't realize about it when I added it last week.